### PR TITLE
Fix test codecoverage

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

This pr is to address the comments on #5858.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by gymad** [comment]
https://github.com/salesagility/SuiteCRM/pull/5858#pullrequestreview-119490226

**Comment by Dillon-Brown** [comment]
Assessed

**Comment by gymad** [comment]
see: https://github.com/salesagility/SuiteCRM/pull/5858#pullrequestreview-120173552

**Comment by gymad** [comment]
more review added to: https://github.com/salesagility/SuiteCRM/pull/5858#pullrequestreview-120173552

File: composer.json [hunk 1]
```diff
@@ -35,8 +35,7 @@
     "jeroendesloovere/vcard": "v1.5",
```
Comment (gymad) [inline composer.json]: see at: https://github.com/salesagility/SuiteCRM/pull/5858#pullrequestreview-120173552

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**